tools/create-element/templates/element/custom-elements-manifest.config.js

```diff
@@ -0,0 +1,5 @@
+import { pfeCustomElementsManifestConfig } from '@patternfly/pfe-tools/custom-elements-manifest.js';
+
+export default pfeCustomElementsManifestConfig({
+  globs: ['<%= tagPrefix %>-.ts'],
```

Comment:
Glob missing `*`

When running `npm run analyze` on a newly created element it will not match the `pfe-*.ts` file

```suggestion
globs: ['<%= tagPrefix %>-*.ts'],
```

Reply:
Nice catch!